examples/expo/package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "react-native-example",
+  "name": "expo-example",
   "version": "0.0.1",
   "private": true,
   "dependencies": {
     "@bitdrift/react-native": "../../dist/react-native",
```

Comment:
Not sure how feasible it would be but it would be super nice if we could depend on the actual files as opposed to the dist variation so that we could automatically pick up changes without having to regenerate dist/

This would let us use this example app and iterate directly on the files